### ISSUE TITLE
Synology + Docker

### ISSUE THREAD

**Issue by brandonjamesmarshall** [kevinvincent/ha-wyzesense#110]
Has anyone gotten this to work on a Synology NAS? I plugged in the USB and I'm not seeing the device in the /dev/ folder

**Comment by mgriffin13** [comment]
nope, see the discussion at https://github.com/kevinvincent/ha-wyzesense/issues/21
